README.md

```diff
@@ -0,0 +1,52 @@
+# COVID-19 Statbot
+## Conflict Plan
+* @here in Slack channel
+* Everyone stops and regroups
+* Differing opinions about how to proceed
+  * Majority rules
+  * If strong feelings, outside opinion (TA, Ryan)
+## Communication Plan
+* Morning standup
+  * 10 minute check-in
+  * Project progress
+  * Each individual's comfort level and engagement
+* Afternoon check-in as needed
+* End of day discussion and merge
+* Encourage taking breaks
+* After hours communication is fine but not required
+* If working after hours, must communicate with whole group about what working on; cannot merge
+* Communication about which files people are working in to avoid merge conflicts
+  * Jira for tracking tickets
+  * Card numbers for branch names
+* Github 
+  * Two people review PR
+    * Slack status?
+  * Maker of PR must be the one to merge
+## MVP
+### Get data
+* Daily number of cases
+  * National
+  * State
+* Daily number of deaths
+  * National
+  * State
+* Number of tests run
+  * National
+  * State
+### Transform data
+* Normalize
+* Turn total number into daily numbers
+* Compare cases/deaths to tests
+### Share data
+* Either Twitter or Slack
+* Tweet daily digest of national numbers
+* Tweet three states w most new cases today
+* Tweet three states w fewest new cases today
+* User can request data by state
+### Stretch
+* Political leanings compared to death rates
+* Political leanings compared to federal supplies granted
+* Transmission rates
+* When shelter in place was implemented for each state/how long they've been under shelter in place
+* Graphing/charting results
+* Simple front end
```

Comment:
I believe we also discussed sending out total cases each morning as another stretch goal.